onnxruntime/contrib_ops/cuda/bert/decoder_attention.cc

```diff
@@ -363,6 +363,9 @@ Status DecoderAttention<T>::ComputeInternal(OpKernelContext* context) const {
 
   return LaunchDecoderAttentionKernel(
           device_prop,
+#ifdef USE_ROCM
```

Comment:
We don't comment these type of thing. There are too many of them are scattering in the CUDA EP make it impractically to do so. Basically, when you see something like this one, they just are useful after hipify. Otherwise, useless for CUDA.